newletter.md

```diff
@@ -0,0 +1,104 @@
+# Linagora's James newsletter
+
+[Linagora](http://linagora.com/) have a team of developpers devoted to the James server. We are already contributing for a few years to the project. This document will try to give you information about what we recently did for the James project, and give hints about the road-map we follow. We will also try to write it regulary.
+
+## Deploying James internally
+
+We now use James as the mail server behind [OpenPaaS](http://open-paas.org/). Thus we deployed it and use it on an every-day base.
+
+![Visual of the James backed OpenPaaS deployment](openpaas.png)
+
+Our deployment is based on the following components :
+ - We enabled IMAP, SMTP and [JMAP](http://jmap.io/) protocols
+ - We use [Cassandra](http://cassandra.apache.org/) for storing e-mails
+ - We rely on [ElasticSearch](https://www.elastic.co/fr/products/elasticsearch) for searching e-mails
+ - We authenticate our users with LDAP
+ - We use [Guice](https://github.com/google/guice/wiki/GettingStarted) for bringing all pieces together.
+ - We plan to bring soon Sieve and ManageSieve part of this deployment as many peaple request filtering.
+
+Let me share a few numbers with you :
+
+ - We handle about 100 users
+ - We have so far a number of 1.500.000 e-mails, that we imported using [IMAP-sync](https://imapsync.lamiral.info/) scripts
+ - We receive around 15.000 incoming e-mails a day
+ - We execute 42.500 IMAP commands a day
+ - We answer 20.000 JMAP requests a day
+
+This deployment help us detecting bugs, and performance issues. We follow the current state of master branch, that we update at midnight 
+with the built image uploaded on [dockerhub](https://hub.docker.com/r/linagora/james-ldap-project/).
+
+## Tracking performance
+
+Everybody wants to read their e-mails fast. Thus the team made performance tracking a priority. And we developed the tools to follow performance.
+
+![Result of 1.000 thunderbird like users](gatling1.png)
+
+We started implementing [Gatling](http://gatling.io/) load testing for the James server :
+
+  - First with a [Gatling JMAP](https://github.com/linagora/james-gatling) implementation.
+  - Then we added a naive SMTP implementation
+  - And finally contributed a [IMAP DSL for Gatling](https://github.com/linagora/gatling-imap)
+
+![Result graph of 1.000 thunderbird like users](gatling2.png)
+
+Running these load tests we :
+
+  - Succeeded to support 1.000 thunderbird like users
+  - However, it turned out we had problems with 10.000 users.
```

Comment:
Please add the limitations of the server (only one server, with cassandra and ES on the same host)